crates/swc_ecma_minifier/src/pass/compute_char_freq.rs

```diff
@@ -19,7 +19,7 @@ pub(crate) struct CharFreqInfo {
 pub(crate) fn compute_char_freq(_m: &Module) -> CharFreqInfo {
```

Comment:
@kdy1 
CharFreqInfo change the `BASE54_DEFAULT_CHARS` order which will break the `is_reserved` function

Reply:
Ah...
Then I think it's fine to remove `CharFreqInfo` for now, as it does not have enough logic to be useful.

Reply:
Would using CharFreqInfo achieve better gzip results?
If so, I'd prefer to keep it. 
I can generate the reserved list at runtime.

Reply:
Not yet at the moment, seems like we need full codegen or smart visitor to make the compression better.